recipe/meta.yaml

```
@@ -22,6 +22,7 @@ requirements:
     - numpy                                  # [build_platform != target_platform]
     - cffi                                   # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
```

nit: Know the bot is doing this, but would use the same quotes here. Either single or double. Personally go for double

```suggestion
    - {{ compiler("c") }}
    - {{ stdlib("c") }}
```